mysql-test/t/secure_log_path-master.opt

```diff
@@ -0,0 +1 @@
+--secure_log_path=$MYSQL_TMP_DIR
```

Comment:
Can you also add a test case for the scenario with default (ie without setting secure_log_path)? where will the general_log_file go?

Reply:
Secure log path defaults to empty, which allows logs to be placed anywhere, as without this change. (the only difference is that now we display a warning about this during startup)

And this scenario is tested by several existing general_log/slow_log tests (main.rewrite_general_log, main.log_tables, main.log_tables_debug, ...)